package.json

```diff
@@ -41,7 +41,8 @@
     "normalize.css": "^8.0.0",
     "rxjs": "^6.5.2",
     "tslib": "^1.9.0",
-    "zone.js": "^0.8.29"
+    "zone.js": "^0.8.29",
+    "node-sass": "^4.13.1"
```

Comment:
Now please remove this change too.

Reply:
> Now please remove this change too.

Removed @Sanji515 but the build failed.

Reply:
I see. Well it should work, it should install `node-sass` as a required dependency while installing other dependencies of `package.json`. Can you try changing this line [here](https://github.com/Cloud-CV/EvalAI-ngx/blob/master/.travis.yml#L20) to just `npm install`?

Reply:
> I see. Well it should work, it should install `node-sass` as a required dependency while installing other dependencies of `package.json`. Can you try changing this line [here](https://github.com/Cloud-CV/EvalAI-ngx/blob/master/.travis.yml#L20) to just `npm install`?

@Sanji515 Sir, it's still failing

Reply:
Also can you please try disabling cache for npm? 
```
cache:
  npm: false
```

Reply:
> Also can you please try disabling cache for npm?
> 
> ```
> cache:
>   npm: false
> ```

Along with keeping the changes in travis.yml?

Reply:
You can revert them.

Reply:
> You can revert them.

It builded successfully @Sanji515

Reply:
@Sanji515 Sir, only the Dashboard's look is left to make it same as that of EvalAI. Should i include that in this PR only?

Reply:
@Kajol-Kumari This pr already have lots of stuff to review.  It will be better to make another PR for that.